includes/transformer/class-post.php

```diff
@@ -275,11 +275,6 @@ protected function get_icon() {
 	 * @return array The Attachments.
 	 */
 	protected function get_attachment() {
-		// Remove attachments from drafts.
-		if ( 'draft' === \get_post_status( $this->item ) ) {
-			return array();
-		}
```

Comment:
This `draft` check is to remove images from the fediverse if a post was first published, federated and then set back to draft mode. We can not remove the check like this, because we have to get sure to remove every media item from the now draft post!

Reply:
We could re-write the code to `Undo` the publish instead of sending an `Update` with a placeholder though.